components/brave_rewards/resources/ui/utils.ts

```diff
@@ -95,7 +95,7 @@ export const getAddresses = (addresses?: Record<Rewards.AddressesType, Rewards.A
 
   sortedArray.forEach((type: Rewards.AddressesType) => {
     const item: Rewards.Address = addresses[type]
-    if (item) {
+    if (item && item.address) {
```

Comment:
`&& item.address` would appear to fix an unrelated legacy issue, if so would it not be better to raise a new ticket and commit this change against that ticket as this way if the issue became a larger problem then it would be more likely an uplift request is accepted